db/schema.rb

```diff
@@ -485,7 +485,6 @@
     t.datetime "updated_at", null: false
     t.bigint "user_id"
     t.datetime "verified_at"
-    t.index ["user_id"], name: "index_email_authorizations_on_user_id"
```

Comment:
@ludwiczakpawel you need to reset your database or manually fix the schema to prevent these schema issues from showing up.

Reply:
reverted.